src/protocol/protocol.yml

```diff
@@ -731,6 +731,9 @@ BrowserContext:
       parameters:
         request: Request
         responseEndTiming: number
+        responseTransferSize: number
```

Comment:
We should use intermediate structs, can be RequestSizes for example and we can update / augment them incrementally, but should be a separate struct.